lib/danger-packwerk/danger_packwerk.rb

```diff
@@ -84,14 +84,23 @@ def check(
 
       packwerk_reference_offenses = PackwerkWrapper.get_offenses_for_files(targeted_files.to_a).compact
 
+      renamed_files = git.renamed_files.map { |before_after_file| before_after_file[:after] }
+
+      # Ignore references that have been renamed
+      packwerk_reference_offenses_to_care_about = packwerk_reference_offenses.reject do |packwerk_reference_offense|
```

Comment:
This part of this PR prevents new packwerk comments for inline comments related a failing packwerk check related to renamed constants.